deployments/ynBTCk-56-v0.0.1.json

```diff
@@ -3,87 +3,94 @@
         "1": {
             "chainId": 1,
             "erc20Address": "0x68589adc7687A23Ff2B06fb032b997f09B44Ed5d",
+            "erc20Implementation": "0xE50AEcb1bBfFABa835366cA8264539C30eD6e1D9",
```

Comment:
Added implementation addresses for all deployment json files to ensure scripts will work for the current deployment json files.

I also. renamed the `ynBTCk` from v0.0.1 to v0.0.2: this is to ensure the scripts will work as is for this deployment json file, else we will have to change the version label to deploy to new networks for ynBTCk.

Reply:
It looks like there's a lot of change in this file, but its mostly only the formatting fix that happened automatically when the deploy script was run.